Documentation/DocuBlocks/Rest/Documents/head_read_document_header.md

```diff
@@ -60,5 +60,3 @@ document's current revision in the *Etag* header.
   ~ db._drop(cn);
 @END_EXAMPLE_ARANGOSH_RUN
 @endDocuBlock
-@endDocuBlock
```

Comment:
Removing the extra end tag does not change the visual result in Swagger UI nor in the docs, so this was probably a mistake.